README.md

```diff
@@ -0,0 +1,150 @@
+## IaaS deployment
+
+   IaaS deployment is a solution which enables to use any provider. It improves flexibility and reduces the provider dependencies by giving the migration very easily to do from one provider to another.
+   The multi-IaaS connector enables to do CRUD operations on different infrastructures on public or private Cloud (AWS EC2, Openstack, VMWare, Docker, etc). It is connected with those infrastructure interfaces in order to manage their lifecycle on virtual machines.
+
+##Build and run
+
+On linux, in the root project directory,
+
+Building the project
+```
+	 $ ./gradlew
+```
+
+Running the application
+```
+	$ ./gradlew run	 
+```
+
+## Manage infrastructures
+
+### List supported infrastructures
+```
+    $ curl -k -X GET http://{IP_ADDRESS}/infrastructures
+```
+
+### Save an infrastructure 
+
+Generic information for saving an infrastructure are :
+- id: the name of the given infrastructure in order to identify it
+- type: the type is cloud nature or the hypervisor used (openstack-nova, aws-ec2, vmware, etc.)
```

Comment:
I think that the exemples are enough in order to describe what is the cloud nature. Don't you think so ?